apps/web/src/components/milestone-logs/recruitment/MilestoneForm.tsx

```diff
@@ -153,7 +160,11 @@ export const MilestoneForm = <T extends MilestoneFormValues>(props: MilestoneFor
                       component={({ field, form }: FieldProps) => (
                         <ReactSelect<MilestoneType>
                           inputId={field.name}
-                          value={outcomeOptions?.find(s => s.id == field.value)}
+                          value={
+                            outcomeGroup?.value === `${STATUS.REFERRAL_ACKNOWLEDGED}`
+                              ? undefined
```

Comment:
this line is to hide the Referal Acknowledged text in the select box for Outcome when adding/ editing a milestone